apps/api/src/schema/websoc.ts

```diff
@@ -75,18 +75,36 @@ const isValidRestrictionCode = (code: string): code is (typeof restrictionCodes)
 export const websocQuerySchema = z.object({
   year: z
     .string({ message: "Parameter 'year' is required " })
-    .length(4, { message: "Parameter 'year' must have length 4" }),
-  quarter: z.enum(terms),
+    .length(4, { message: "Parameter 'year' must have length 4" })
+    .openapi({ description: "The year of the WebSoc section", example: "2025" }),
```

Comment:
Adding "WebSoc" every time is clunky and it should be clear from surrounding context that this query parameter is undeniably for WebSoc. You should also look closely at the results of this endpoint and determine where "course" or "section" is the correct term. I will not repeat that comment but it extends throughout. When #73 is resolved, this schema definition in particular will become obsolete.